components/QRCode.tsx

```diff
@@ -1,22 +1,22 @@
 import { LinearGradient } from "expo-linear-gradient";
 import { View } from "react-native";
 import QRCodeLibrary from "react-native-qrcode-svg";
-
+import { SHADOWS } from "~/lib/constants";
 
 function QRCode({ value }: { value: string }) {
-    return (
-        <LinearGradient
-            className="p-2 rounded-2xl flex items-center justify-center"
-            colors={['#FFC453', '#FFE951']}
-            start={[0, 1]}
-            end={[1, 0]}
-            style={{ borderRadius: 28, elevation: 2 }}
-        >
-            <View className="flex items-center justify-center p-3 rounded-3xl bg-white w-96 h-96">
-                <QRCodeLibrary value={value} size={300} />
-            </View>
-        </LinearGradient>
-    );
+  return (
+    <LinearGradient
+      className="p-2 rounded-3xl flex items-center justify-center"
+      colors={["#FFC453", "#FFE951"]}
+      start={[0, 1]}
+      end={[1, 0]}
+      style={SHADOWS.medium}
```

Comment:
this shadow doesn't work for me

Reply:
Yeah for me too, I think LinearGradient doesn't support shadows even in style, the solution would be to wrap it in a View again, but I thought it would complicate things

Reply:
I think it's ok just remove it. I am just pointing out you added dead code

Reply:
This is not completely unused, because in android the elevation is used and adds a shadow. Should we remove the shadow entirely otherwise?